app/views/homepage/_chevron_card.html.erb

```diff
@@ -0,0 +1,31 @@
+<%
+  element ||= :li
+
+  tracking_attributes = {
+    track_action: "topicsLink",
+    track_category: "homepageClicked",
+    track_dimension_index: 29,
+    track_dimension: raw(item[:title]),
+    track_label: item[:href],
+    track_value: 1,
+  }
+
+  link = capture do
+    link_to(raw(item[:title]), item[:href], {
+      class: "govuk-link govuk-link--no-visited-state chevron-card__link",
+      data: tracking_attributes,
+    })
+  end
+%>
+
+<%= content_tag(element, class: "chevron-card") do %>
```

Comment:
Should this be `app-c-chevron-card`?

Reply:
I'd say not yet as it's not a proper component - I've tried to make it as component-like as possible, but at the moment I don't think it needs the prefix.

Reply:
I can live with that. Resolving.